Description: Fix sudo support
Forwarded: https://git.reviewboard.kde.org/r/120380/
Author: Jonathan Riddell <jriddell@ubuntu.com>
Last-Update: 2014-26-09
Index: kdesu/CMakeLists.txt
===================================================================
--- kdesu.orig/CMakeLists.txt	2015-01-28 10:50:51.818137441 +0100
+++ kdesu/CMakeLists.txt	2015-01-28 10:50:51.790138562 +0100
@@ -63,4 +63,5 @@
   DESTINATION ${KF5_INCLUDE_INSTALL_DIR} COMPONENT Devel
 )
 
+message("Note: pass -DKDESU_USE_SUDO_DEFAULT=true to use sudo instead of su")
 feature_summary(WHAT ALL   FATAL_ON_MISSING_REQUIRED_PACKAGES)
Index: kdesu/src/CMakeLists.txt
===================================================================
--- kdesu.orig/src/CMakeLists.txt	2015-01-28 10:50:51.818137441 +0100
+++ kdesu/src/CMakeLists.txt	2015-01-28 10:50:51.806137922 +0100
@@ -30,6 +30,7 @@
 generate_export_header(KF5Su BASE_NAME KDESu EXPORT_FILE_NAME kdesu/kdesu_export.h)
 add_library(KF5::Su ALIAS KF5Su)
 
+target_compile_definitions(KF5Su PRIVATE -DKDESU_USE_SUDO_DEFAULT="${KDESU_USE_SUDO_DEFAULT}")
 target_include_directories(KF5Su PRIVATE ${CMAKE_CURRENT_BINARY_DIR}/..)
 
 # Apps must include <KDESu/File> or <kdesu/file.h>
